README.md

```diff
@@ -52,27 +58,28 @@ const todosReducer = (state: Todo[] = [], action: Action) =>
     // handle cases as pure functions instead of switch statements
     ADD_TODO: ({ id, text }) => [...state, { id, text, completed: false }],
     TOGGLE_TODO: ({ id }) =>
-      state.map(
-        todo =>
-          todo.id === id ? { ...todo, completed: !todo.completed } : todo
+      state.map(todo => todo.id === id
+        ? { ...todo, completed: !todo.completed }
+        : todo
       ),
     // handles the rest; if not provided, cases must be exhaustive 
-    default: a => state // a===action. Useful for curried version. 
+    default: a => state // a === action. Useful for curried version
   });
 ```
 
- `action` can be omitted. In that case the result of match is a function
+`action` can be omitted; in that case the result of match is a function:
 
 ```ts
 const getIdFromAction = Action.match({
   ADD_TODO: ({ id}) => id,
   TOGGLE_TODO: ({ id }) => id,
-  default: a => `${a.tag} doesn't have id`
+  default: a => undefined,
```

Comment:
Should this throw an exception referencing `a.type`? It seems very odd to return a `string` which could be confused for an ID.

Reply:
id is a string type. So i guess throwing an exception would be a better example

Reply:
Yes I like it more 👍